src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        String dash = "____________________________________________________________";
+        String greeting1 = "Pai Kia Bot: Eh harlo! Call me Pai Kia Bot.";
+        String greeting2 = "Pai Kia Bot: What you want?";
+        String greeting3 = "Pai Kia Bot: I will keep a list of your responses ah";
+        String farewell = "Pai Kia Bot: Leave so soon ah? Limpeh sleep first, if got no issue don't disturb me.";
+
+        ArrayList<CheckList> inputList = new ArrayList<>();
+
+        System.out.println(dash);
+        System.out.println(greeting1);
+        System.out.println(greeting2);
+        System.out.println(dash);
+        System.out.print("Your response: ");
+        String input = sc.nextLine();
+        System.out.println("");
+
+        while (!input.equals("bye")) {
```

Comment:
I think that using `switch` and `case` would be a lot more readable than `if else` statements here

Reply:
May I also suggest switching to a more OOP-oriented code. Perhaps you could create a Task object and extend it from there for each of the tasks and allow each task to handle its own inputList?